### ISSUE TITLE
Are file names universally unique?

### ISSUE THREAD

**Issue by mradlmaier** [coomar2841/image-chooser-library#129]
Are file names universally unique?

Or is it possible that the image-chooser-library generates the same file names on different android devices?

**Comment by coomar2841** [comment]
No, They can't be unique across all devices. It's just a timestamp.

**Comment by mradlmaier** [comment]
Is it possible to make them have unique file names, by using UUID instead
of timestamps?
This would make Synchronisation of files across devices easier.

Kumar Bibek <notifications@github.com> schrieb am Sa., 14. Nov. 2015 um
19:01:

**Comment by coomar2841** [comment]
Yes, sure. Will add this for the next version.

**Comment by mradlmaier** [comment]
Great!

Kumar Bibek <notifications@github.com> schrieb am Sa., 14. Nov. 2015 um
19:09:
